fe2-android/app/schemas/com.github.dedis.popstellar.repository.database.AppDatabase/3.json

```diff
@@ -1,515 +0,0 @@
-{
```

Comment:
why do we not need this anymore ?

Reply:
This was for a cleanup, that schema was unused but I forgot to delete it, the up-to-date version is the 4th